src/middlewared/middlewared/etc_files/syslog-ng/syslog-ng.conf.mako

```diff
@@ -90,6 +90,10 @@ source s_tn_auditd {
   unix-stream("/var/run/syslog-ng/auditd.sock" create-dirs(yes) perm(0600));
 };
 
+source s_tn_audit_handler_errors {
+  unix-stream("${AUDIT_HANDLER_ERROR_SOCKET}" create-dirs(yes) perm(0600));
```

Comment:
Why can't this just go to normal syslog? We're avoiding normal syslog pipe for audit messages because we want to enforce access control on processes that can actually write the audit messages for the auditd DB.